.circleci/config.yml

```diff
@@ -38,6 +43,14 @@ jobs:
           command: |
             echo "$GCLOUD_SERVICE_KEY" > credentials.json
             echo "export GOOGLE_APPLICATION_CREDENTIALS=\"$(realpath credentials.json)\"" >> $BASH_ENV
+            google-cloud-sdk/bin/gcloud auth activate-service-account --key-file=credentials.json
+      - run:
+          name: "Set up k8s cluster"
+          command: |
+            kubectl version --client
+            google-cloud-sdk/bin/gcloud config set project styx-oss-test
```

Comment:
We don't need to change default settings. All gcloud commands accept `--project`.

Reply:
👍